### PR TITLE
Codespell: config, action and typos fixed

### PR DESCRIPTION
**Pull request by yarikoptic**
I was nearby, so decided to "stop by" and say "Hi"

### CODE REVIEW

File: extra/mean_sq_deviations.md [hunk 1]
```diff
@@ -66,8 +66,8 @@ We can find this location by transforming the formula in {eq}`sse_c` above
 into a formula for the *gradient* of the line that {eq}`sse_c` represents.
 This is *taking the derivative*.  When the *derivative* of equation
 {eq}`sse_c` is equal to zero, it means the *gradient* of 
-{eq}`sse_c` is 0, and this is true when we are at a peak or a trough of
-{eq}`sse_c`.  We want the trough.
+{eq}`sse_c` is 0, and this is true when we are at a peak or a through of
```
Comment (matthew-brett) [inline extra/mean_sq_deviations.md]: Oops - "trough" is correct !

**Review by matthew-brett — COMMENTED** [review]
Thanks much for this - but - any idea how to tell the thing not to second guess "trough" there?

**Comment by yarikoptic** [comment]
Thanks for catching. I added it to skips in the config, and reran the fixup.

File: permutation/population_permutation.Rmd [hunk 2]
```diff
@@ -80,10 +80,10 @@ Leave voters into another array, called `leave_ages`.
 # Extract Remain and Leave voters into their separate own data frames.
 # As above, we are just using Pandas here to get arrays for the
 # Remain and Leave voter ages.
-remainers = brexit_df[brexit_df['brexit_vote'] == 'Remain']
+remainders = brexit_df[brexit_df['brexit_vote'] == 'Remain']
```
Comment (yarikoptic) [inline permutation/population_permutation.Rmd]: but we should also ignore these `remainers` probably in all of their forms right?

Comment (matthew-brett) [inline permutation/population_permutation.Rmd]: Ah - sorry - also `remainers` is correct here.

Comment (yarikoptic) [reply permutation/population_permutation.Rmd]: I skipped them too now!

**Review by matthew-brett — COMMENTED** [review]
Oops - spotted another not-for-correction word

File: permutation/population_permutation.Rmd (hunk 2 above)
Comment (yarikoptic) [reply permutation/population_permutation.Rmd]: yeap -- we saw it in parallel, fixed already, sorry for the sloppy original submission ;-)

**Comment by matthew-brett** [comment]
Excellent - thanks again.

**Pull request merged**